src/jit/lower.h

```diff
@@ -55,27 +55,63 @@ class Lowering : public Phase
             LowerNode(newNode);
         }
     }
+    void LowerRange(GenTree* firstNode, GenTree* lastNode)
+    {
+        for (GenTree* node = firstNode; node != lastNode->gtNext; node = node->gtNext)
+        {
```

Comment:
FWIW, the intended pattern for things like this is as follows:
- Create a scratch `Range` using `LIR::EmptyRange()`
- Add whatever code you need to add to the scratch `Range`
- Insert the `Range` into its destination

See the definition and uses of `DecomposeRange` for examples.